package.json

```diff
@@ -4,7 +4,10 @@
   "homepage": "https://TomaszDziedzicITA.github.io/redactlePL",
   "private": true,
   "dependencies": {
-    "@mui/material": "^5.10.5",
+    "@emotion/react": "^11.10.4",
```

Comment:
I think we do not need emotion as we are using styled components instead. Please verify if it can be removed.

Reply:
It is needed for material icons. Documentations specifise:
The @mui/icons-material package depends on @mui/material, which requires Emotion packages.

If you don't use Material UI in your project yet, install the icons package with npm install @mui/icons-material @mui/material @emotion/styled @emotion/react.
See the [Installation](https://mui.com/material-ui/getting-started/installation/) page for additional docs about how to make sure everything is set up correctly.